### ISSUE TITLE
Chat command authority

### ISSUE THREAD

**Issue by Eastrall** [Eastrall/Rhisis#96]
At the moment, we check the chat command authority in each command. As discussed in PR #95 it would be nice to add an extra parameter on the `ChatCommand` attribute so we can specify the required authority to execute a command.

```cs
[ChatCommand("/myCommand", AuthorityType.GameMaster]
public static void OnMyCommand(IPlayerEntity player, string[] parameters)
{
     // ...
}
```

**Comment by Eastrall** [comment]
Done in PR #95 by @johmarjac.

**Comment by johmarjac** [comment]
@Eastrall This has been implemented using #95 and can be closed I guess.
